micromanager_gui/_gui_objects/_mm_configuration_widget.py

```diff
@@ -0,0 +1,51 @@
+from qtpy import QtWidgets as QtW
+
+
+class MMConfigurationWidget(QtW.QWidget):
+    """
+    Contains the following objects:
+
+    cfg_groupBox: QtW.QGroupBox
+    cfg_LineEdit: QtW.QLineEdit
+    browse_cfg_Button: QtW.QPushButton
+    load_cfg_Button: QtW.QPushButton
+    properties_Button: QtW.QPushButton
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.setup_gui()
+
+    def setup_gui(self):
+
+        self.main_layout = QtW.QGridLayout()
+        # cfg groupbox in widget
+        self.cfg_groupBox = QtW.QGroupBox()
+        self.cfg_groupBox.setTitle("Micro-Manager Configuration")
+        self.main_layout.addWidget(self.cfg_groupBox, 0, 0)
+        self.setLayout(self.main_layout)
+
+        # define camera_groupBox layout
+        self.cfg_groupBox_layout = QtW.QGridLayout()
+
+        # add to cfg_groupBox layout:
+        self.cfg_LineEdit = QtW.QLineEdit()
```

Comment:
```suggestion
        self.cfg_LineEdit = QtW.QLineEdit()
        self.cfg_LineEdit.setPlaceHolderText("MMConfig_demo.cfg")
```

Reply:
this placeholder got lost in the move away from `ui` file.

Reply:
I'll put it back in the widget.